docs/releases/5.2.txt

```diff
@@ -165,7 +165,7 @@ File Uploads
 Forms
 ~~~~~
 
-* ...
+* Added :class:`~django.forms.SearchInput` form widget that renders as ``<input type="search" ...>``.
```

Comment:
```suggestion
* The new :class:`~django.forms.SearchInput` form widget renders as
  ``<input type="search" ...>``. This is to be used for entering search queries.
```
Our release notes are written in present tense 👍

Reply:
Done! Thanks for the review